modules/pam_shells/pam_shells.c

```diff
@@ -139,6 +139,7 @@ static int perform_check(pam_handle_t *pamh)
  #endif
 
     if (retval) {
+	pam_syslog(pamh, LOG_NOTICE, "user has an invalid shell '%s'", userShell);
```

Comment:
The log message should start with a capital letter to be consistent with other messages from this module.